client/web/reporter/src/components/Common/Prefilter.vue

```diff
@@ -67,36 +67,67 @@
                       :icon="['fas', 'filter']"
                       size="sm"
                     />
-                  </b-button>
-                </b-input-group-prepend>
+                  </b-button> 
+                </template>
 
-                <b-form-select
-                  v-model="group.args[0].args[argIndex].args[0].args[0].symbol"
-                  :options="columns"
-                  text-field="label"
-                  value-field="name"
-                  style="max-width: 33%;"
-                  @change="setType(groupIndex, argIndex)"
-                >
-                  <template #first>
-                    <b-form-select-option
-                      value=""
-                    >
-                      {{ $t('general:label.none') }}
-                    </b-form-select-option>
-                  </template>
-                </b-form-select>
-
-                <b-form-select
-                  v-model="group.args[0].args[argIndex].args[0].ref"
-                  :options="operators"
-                  style="max-width: 60px;"
-                />
+                <template v-if="group.args[0].args[argIndex].args[0].args[0].value && getColumnData(group.args[0].args[argIndex].args[0].args[1]).multivalue">
+                    <b-form-input
+                      v-model="group.args[0].args[argIndex].args[0].args[0].value['@value']"
+                      :placeholder="$t('builder:value')"
+                    />
 
-                <b-form-input
-                  v-model="group.args[0].args[argIndex].args[0].args[1].value['@value']"
-                  :placeholder="$t('builder:value')"
-                />
+                    <b-form-select
+                      v-model="group.args[0].args[argIndex].args[0].ref"
+                      :options="getOperators(getColumnData(group.args[0].args[argIndex].args[0].args[1]))"
+                      style="max-width: 120px;"
+                    />
+
+                    <b-form-select
+                      v-model="group.args[0].args[argIndex].args[0].args[1].symbol"
+                      :options="columns"
+                      text-field="label"
+                      value-field="name"
+                      @change="setType(groupIndex, argIndex, group.args[0].args[argIndex].args[0].args[1].symbol, group.args[0].args[argIndex].args[0].args[0].value['@value'])"
```

Comment:
Instead of passing in symbol and value into the function, you can compute them inside it